app/client/rest/tracking.ts

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {DeviceEventEmitter, Platform} from 'react-native';
+
+import {Events} from '@constants';
+import {t} from '@i18n';
+import {setServerCredentials} from '@init/credentials';
+import {getFormattedFileSize} from '@utils/file';
+import {logDebug, logInfo} from '@utils/log';
+import {semverFromServerVersion} from '@utils/server';
+
+import * as ClientConstants from './constants';
+import ClientError from './error';
+
+import type {APIClientInterface, ClientHeaders, ClientResponseMetrics, RequestOptions} from '@mattermost/react-native-network-client';
+
+type UrlData = {
+    count: number;
+    metrics?: ClientResponseMetrics;
+}
+
```

Comment:
This is done in a future PR as I mentioned in my previous comment, so for this PR it will remain as is

Reply:
> For example you'll see entry and entry-additional in this PR but in a future PR I have almost ready to go it will change to the following labels (this labels will be well defined in a type definition):

Thanks, I missed it.